control.go

```diff
@@ -44,3 +48,15 @@ func (a *App) drawFullAction(action string) {
 	pos := NewPos(ScreenWidth/2, ScreenHeightScene)
 	a.drawDefaultText(action, pos, AlignCenter, ControlActionColor)
 }
+
+func (a *App) processControlInputs() {
+	if rl.IsMouseButtonPressed(rl.MouseButtonLeft) { // TODO missing check action / control selected
+		if a.MouseIsInto(a.scene.sceneViewPort) {
```

Comment:
I was not considering the scene viewport differs from one scene to another. In my brain, it is fixed to `Rectangle{Pos: {X: 0, Y: 0}, Size: {W: 320, H: 144}}`. It is a way to differentiate if the mouse click occurred in the scene viewport instead of the control panel.

Reply:
Done!